homeassistant/components/sensor/dominionenergy.py

```diff
@@ -0,0 +1,49 @@
+"""Support for Washington Dominion Energy."""
+
+from homeassistant.util import Throttle
+from homeassistant.helpers.entity import Entity
+
+
```

Comment:
The configuration needs to run through the validator -> https://home-assistant.io/developers/code_review_platform/#3-configuration